### PR TITLE
Jetpack Settings: Speed up your site

### PR DESCRIPTION
**Pull request by elibud**
**Fixes** #8568 

This PR adds two new Site Settings under the section Speed up your site: 
- Lazy Load Images 
- Serve Images from our servers

**To test:**
1. Test that the new section under writing is only present for Jetpack sites.
2. Test that updating the settings in the app is reflected in `/sites/SITE_ID/jetpack/modules` `v 1.1`

newjetpacksettings

cc: @planarvoid

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/JetpackSpeedUpSiteSettingsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+    // MARK: - Model
+
+    fileprivate func reloadViewModel() {
+        handler.viewModel = tableViewModel()
+    }
+
+    func tableViewModel() -> ImmuTable {
+
+        let serveImagesFromOurServers = SwitchRow(title: NSLocalizedString("Server images from our servers",
```
Comment (koke) [inline WordPress/Classes/ViewRelated/Blog/JetpackSpeedUpSiteSettingsViewController.swift]: Typo: both string and comment should say `Serve`

File: WordPress/Classes/ViewRelated/Blog/JetpackSpeedUpSiteSettingsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                                       onChange: self.lazyLoadImagesValueChanged())
+
+        return ImmuTable(sections: [
+            ImmuTableSection(
+                headerText: "",
+                rows: [serveImagesFromOurServers],
+                footerText: NSLocalizedString("Jetpack will optimize your images and serve them from the server " +
+                                              "location nearest to your visitors. Using our global content delivery " +
+                                              "network will boost the loading speed of your site.",
+                                              comment: "Footer for the Server images from our servers setting")),
```
Comment (koke) [inline WordPress/Classes/ViewRelated/Blog/JetpackSpeedUpSiteSettingsViewController.swift]: Another `Server` in the comment

File: WordPress/Classes/ViewRelated/Blog/JetpackSpeedUpSiteSettingsViewController.swift [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+
+            ImmuTableSection(
+                headerText: "",
+                rows: [lazyLoadImages],
+                footerText: NSLocalizedString("Improve your site's speed by only loading images visible on the screen. " +
+                                              "New images will load just before they scroll into view. This prevents " +
+                                              "viewers from having to download all the images on a page all at once, " +
+                                              "even ones they can't see.",
+                                              comment: "Footer for the Server images from our servers setting")),
```
Comment (koke) [inline WordPress/Classes/ViewRelated/Blog/JetpackSpeedUpSiteSettingsViewController.swift]: Another `Server` in the comment

**Comment by elibud** [comment]
Thanks for the review @koke I've fixed those, will merge when green.

**Pull request merged**